ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -102,6 +107,113 @@ def _get_table_description(schema: str, table: str, inspector: Inspector) -> str
     return description
 
 
+def get_dbt_local(config) -> Optional[Tuple[str, str]]:
+    try:
+        if config.dbtConfig.dbtCatalogFilePath is not None:
+            with open(
+                config.dbtConfig.dbtCatalogFilePath, "r", encoding="utf-8"
+            ) as catalog:
+                dbt_catalog = catalog.read()
+        if config.dbtConfig.dbtManifestFilePath is not None:
+            with open(
+                config.dbtConfig.dbtManifestFilePath, "r", encoding="utf-8"
+            ) as manifest:
+                dbt_manifest = manifest.read()
+        return json.loads(dbt_catalog), json.loads(dbt_manifest)
+    except Exception as exc:
+        logger.debug(traceback.print_exc())
+        logger.debug(f"Error fetching dbt files {repr(exc)}")
```

Comment:
Thanks for separating these methods. Let's make the logging messages a bit more clear, such as `Error fetching db files from local`, or `Error fetching dbt files from HTTP....`.

Also, Instead of `traceback.print_exc()`, we should use `traceback.format_exc()` when passing it inside logger.debug